core/src/main/java/feign/FeignException.java

```diff
@@ -168,6 +168,10 @@ static FeignException errorReading(Request request, Response response, IOExcepti
   }
 
   public static FeignException errorStatus(String methodKey, Response response) {
+    return errorStatus(methodKey, response, null, null);
+  }
+  public static FeignException errorStatus(String methodKey, Response response,
+                                           Integer maxBodyBytesLength, Integer maxBodyCharsLength) {
```

Comment:
This method is never invoked by feign.

So what would be the point of the change.

I'm sure I'm missing something

Reply:
@velo it's that issue https://github.com/OpenFeign/feign/issues/1606, need for me, as a feign client.

With these parameters, my exception body will be full information in error decoder handler.

Reply:
@coungard what I'm not understanding is how.

That new method is never invoked by feign... how is that going to change anything?

Reply:
@velo the result can be clearly seen on the test -> FeignExceptionTest::lengthOfBodyExceptionTest (143)

Reply:
@velo Sorry, but isn't that enough? The code is called by Feign by default (null, null), but for us it will be overridden parameters:
![изображение](https://user-images.githubusercontent.com/32792990/193588589-e1501d60-ffcf-448b-af9c-f1afce6f3c89.png)

Reply:
ow, I see now, my bad